_data/whats-new.yml

```diff
@@ -4,8 +4,179 @@ description: |
+    topic in the _Cloud Guide_:<br/>- Added additional validation and tasks to address
+    common Support issues<br/>- Added additional links to supporting information and
+    instructions<br/>- Added a new "Upgrade best practices" subsection to provide
+    specific upgrade steps
+  versions: 2.x
+  type: Major update
+  date: October 22, 2019
+  link: https://github.com/magento/devdocs/pull/5703
+- description: Updated the [Cloud launch modes](https://devdocs.magento.com/guides/v2.2/cloud/docker/docker-config.html)  documentation
+    to configure Varnish as a http-cache-host so that Varnish purges with a Magento
+    cache clean.
+  versions: 2.x
+  type: Technical changes
+  date: October 22, 2019
+  link: https://github.com/magento/devdocs/pull/5721
+- description: Added the documentation for the [DynamicRowsRecord](https://devdocs.magento.com/guides/v2.2/ui_comp_guide/components/ui-dynamicrows-record.html)
+    UI component to the _UI Components Guide_.
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: October 21, 2019
+  link: https://github.com/magento/devdocs/pull/5634
+- description: Updated the [Plugins (Interceptors)](https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-manage.html)
+    topic in the _Frontend Developer Guide_ to clarify that these plugins are for
+    updating layouts. Also, removed incorrect information and clarified descriptions
+    and examples.
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: October 21, 2019
+  link: https://github.com/magento/devdocs/pull/5547
+- description: Changes to the status of Magento Shipping are now described in the
+    [Magento Commerce 2.3.3 Release Notes](https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-3-commerce.html)  and
+    the [Magento Open Source 2.3.3 Release Notes](https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-3-open-source.html).
+  versions: 2.3.3
+  type: Major update
+  date: October 20, 2019
+  link: https://github.com/magento/devdocs/pull/5807
+- description: Updated the [Magento Commerce Release Notes](https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-3-commerce.html)
+    to add information about a known issue with method chaining that has been identified
+    in the 2.3.3 code base and the patch that was released on October 18 to address
+    this issue.
+  versions: ''
+  type: Major update
+  date: October 18, 2019
+  link: https://github.com/magento/devdocs/pull/5800
+- description: Moved the `CategoryInterface` attributes to a separate [topic](https://devdocs.magento.com/guides/v2.3/graphql/product/category-interface.html)
+    in the _GraphQL Developer Guide_.
+  versions: 2.3.x
+  type: New topic
+  date: October 16, 2019
+  link: https://github.com/magento/devdocs/pull/5718
+- description: Added an "Errors" section to the [changeCustomerPassword mutation](https://devdocs.magento.com/guides/v2.3/graphql/mutations/change-customer-password.html)
```

Comment:
I am not sure you reference doc sections in other line items in quotes... check for consistency.

Reply:
Also consistency in ending punctuation.